recipes/pyjnius/activate.bat

```diff
@@ -0,0 +1,6 @@
+SET "PYJNIUS_JAR_BACKUP=%PYJNIUS_JAR%"
+SET "PYJNIUS_JAR=%CONDA_PREFIX%\share\pyjnius\pyjnius.jar"
+SET "JDK_HOME_BACKUP=%JDK_HOME%"
+SET "JDK_HOME=%JAVA_HOME%"
```

Comment:
Would be curious to see if we can just drop the `%JDK_HOME%` stuff given we didn't need it on Unix and it seems like we have a better understanding of the problem now.

Reply:
The PyJNIus doc states that we should set it, so I'd just leave it in there.
http://pyjnius.readthedocs.io/en/latest/installation.html#installation-for-windows